### PR TITLE
[coturn] initial integration

### PR DESCRIPTION
**Pull request by 0x34d**
Hello @misi 
For More Security Related issues in the @coturn interface, The ```coturn``` project can in deployed in ```oss-fuzz``` infrastructure.
I and ```oss-fuzz``` team would like to hear your thoughts on integration.

Signed-off-by: 0x34d <ajsinghyadav00@gmail.com>

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
We are happy to accept this project if upstream wants it integrated.

**Comment by 0x34d** [comment]
@bradleythughes ping.

**Comment by 0x34d** [comment]
@ggarber @bradleythughes @tyranron ping. 
Any Comment?

File: projects/coturn/build.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# limitations under the License.
+#
+################################################################################
+
+cd fuzz
+make all
+
+cp FuzzStun $OUT/FuzzStun
+cp FuzzStunClient $OUT/FuzzStunClient
+cp input/FuzzStun_seed_corpus.zip $OUT/FuzzStun_seed_corpus.zip
```
Comment (jonathanmetzman) [inline projects/coturn/build.sh:23]: Please download the zips from some place else.

**Review by jonathanmetzman — CHANGES_REQUESTED** [review]
lgtm other than the zips, please fix

File: projects/coturn/project.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+homepage: "https://github.com/coturn/coturn"
+language: c
+primary_contact: "misi@majd.eu"
+vendor_ccs:
+  - "mom040267@gmail.com"
```
Comment (jonathanmetzman) [inline projects/coturn/project.yaml:5]: Upstream is auto_cc, you are either.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

**Pull request merged**